pkg/cmd/pulumi/destroy.go

```diff
@@ -130,7 +132,12 @@ func newDestroyCmd() *cobra.Command {
 			}
 
 			proj, root, err := readProject()
-			if err != nil {
+			if err != nil && errors.Is(err, workspace.ErrProjectNotFound) {
+				logging.Warningf("failed to find current Pulumi project, continuing using stack %v from backend %v",
```

Comment:
```suggestion
				logging.Warningf("failed to find current Pulumi project, continuing continuing with an empty project using stack %v from backend %v",
```

Reply:
Applying the suggestion & merging after tests complete.